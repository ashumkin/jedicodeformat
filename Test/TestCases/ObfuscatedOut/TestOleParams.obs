unit testoleparams;interface implementation uses comobj;procedure wordup;var msword
:variant;begin msword:=createoleobject('Word.Basic');msword.appshow;msword.filenew
;msword.insert('Foo');msword.lineup(2,1);msword.texttotable(convertfrom:=2,numcolumns
:=3);end;procedure w_app;var wordapp:variant;begin wordapp:=createoleobject('Word.Application'
);wordapp.documents.open('foo',,,,'bar');end;end. 