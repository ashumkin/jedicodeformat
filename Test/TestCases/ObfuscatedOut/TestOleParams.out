unit testoleparams;

interface

implementation

uses comobj;

procedure wordup;
var
  msword: variant;
begin
  msword := createoleobject('Word.Basic');
  msword.appshow;
  msword.filenew;
  msword.insert('Foo');
  msword.lineup(2, 1);
  msword.texttotable(convertfrom := 2, numcolumns := 3);
end;

end. 